README.md

```diff
@@ -61,6 +62,38 @@ The SDKs provide the following functionality:
 - Access token can be automatically and continuously refreshed
 - Redirect callback is invoked after login or register
 
+## Automated testing
+The [Playwright](https://playwright.dev/docs/intro) end-to-end tests verify the proper functionality of several authentication and authorization endpoints in the FA SDK.
+
+Prerequisites
+- Ensure Playwright has been installed as a dependency 
+- On a seperate server run the FA SDK consuming quickstart application.
+
+Configuration
+The Playwright configuration (playwright.config.ts) includes settings for running tests in parallel, specifying browser environments, and defining the base URL the tests will perform on.
+
+Running the Tests
+Prior to running the tests you will want to check the Server Command that is used to start up a local instance of the FA SDK consuming quickstart application. Additionally you will want to note the port number this application will run on.
```

Comment:
<img width="568" alt="image" src="https://github.com/FusionAuth/fusionauth-javascript-sdk/assets/49047386/53d86529-854b-4047-bdba-01d669f65f79">

It looks like markdown isn't linebreaking like you'd expect here. Looks like it needs an extra new line.